Library/Formula/nq.rb

```diff
@@ -0,0 +1,21 @@
+class Nq < Formula
+  desc "Unix command-line queue utility"
+  homepage "https://github.com/chneukirchen/nq"
+  url "https://github.com/chneukirchen/nq/archive/v0.1.tar.gz"
+  sha256 "e0962a5e6a2cbf799bba7a79af22c40d21e5a80605df42c8bba37d3d8efb1793"
+
+  head "https://github.com/chneukirchen/nq.git"
+
+  depends_on :macos => :yosemite
```

Comment:
It is known-not-working on 10.9 through 10.8: https://github.com/chneukirchen/nq/issues/12 and https://github.com/chneukirchen/nq/issues/6

I have no idea how to even test this on 10.3, but maybe that has those functions. It would be surprising, because they appear in POSIX 2008, and 10.3 was released in... 2005?